sequins.go

```diff
@@ -150,7 +168,14 @@ func (s *sequins) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	key := strings.TrimPrefix(r.URL.Path, "/")
-	res, err := s.index.Get(key)
+
+	currentIndex := s.indexMonitor.AcquireRef()
+	if currentIndex == nil {
+		log.Fatal("Tried to make HTTP request before initializing the index") 
+	}
```

Comment:
Rather than fataling, maybe we could grab the RWMutex on the initial load, so that we know for sure this can't happen?